src/main/java/org/icatproject/core/entity/DataPublicationType.java

```diff
@@ -0,0 +1,74 @@
+package org.icatproject.core.entity;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+@Comment("A type of data publication, for example, whole investigation, user-selected datasets/files")
```

Comment:
I'm unsure: maybe one could add a hint that this is most likely only relevant for facility internal purposes and that facilities need to define their own classification scheme.

Reply:
I've extended the comment to clarify this.